new/LIT5843ShortPoemMary.xml

```diff
@@ -53,11 +52,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     </teiHeader>
     <text>
         <body>
-            <div type="bibliography">
+             <div type="bibliography">
                 <listRelation>
                     <relation name="lawd:hasAttestation" active="LIT5843ShortPoemMary" passive="Neukirchen005"/>
                 </listRelation>
-            </div><!---->
+                <div type="textpart" corresp="Neukirchen005#ms_i3" xml:lang="gez">
```

Comment:
`<div type="textpart">` should be contained in a `<div type="edition">`

Reply:
Ok